doc/distrib/NodeHelpFiles/DSRevitNodesUI.CeilingTypes.md

```diff
@@ -0,0 +1,6 @@
+## In Depth
+Allows a selection of a single Ceiling type from all the ceiling types available in the current document (file). This node is useful with the Ceiling.ByOutlineTypeAndLevel node.
```

Comment:
Please include the node name in in-depth descriptions, for example, `Ceiling Types` allows a selection of...
Also if possible, please include a descriptions of examples, starting with "In the example below..."